docs/bnb-smart-chain/developers/node_operators/archive_node.md

```diff
@@ -13,10 +13,11 @@ Simply speaking, an archive node is a full node running with an additional speci
 
 Developers are limited to querying the limited recent blocks to check the balance of an address and the state of a smart contract with a full node. It is hard to get all what they want as the blockchain is moving forward at the same time, while they can query any block at a specific point in time with an archive node.
 Archive nodes are used by various applications on the blockchain for challenging use cases, including but not limited to the followings:
-* Automatic trading system needs historical data to optimize trading model
```

Comment:
<img width="773" alt="image" src="https://github.com/user-attachments/assets/c8306812-4842-44ef-9dfd-60d86e86db0f">

not expected